### PR TITLE
IGAPP-286: Test and fix deliveries

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by Taggotty — APPROVED** [review]

File: .circleci/src/workflows/commit.yml [hunk 1]
```diff
@@ -1,49 +1,7 @@
 unless: << pipeline.parameters.api_triggered >>
```
[the remark this answers is not in the document]
Comment (klinzo) [reply .circleci/src/workflows/commit.yml]: Yep, sorry! Forgot I already marked it as ready for review.

File: .circleci/src/jobs/build_ios.yml [hunk 2]
```diff
@@ -63,17 +63,17 @@ steps:
             command: bundle exec fastlane build version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE} build_config_name:integreat
             working_directory: native/ios
         - run:
-            name: Rename sourcemap
-            command: mv index.ios.bundle.map output/integreat.ios.bundle.map
-            working_directory: native/ios
+            name: Create sourcemap
```
Comment (maxammann) [inline .circleci/src/jobs/build_ios.yml:66]: I'm not sure whether we want to additinally run `react-native bundle` this takes about 1-2 minutes and slows down the CI. We can also just pass a parameter to the xcode script which then creates a source map.

**Review by maxammann — CHANGES_REQUESTED** [review]

File: .circleci/src/jobs/build_ios.yml (hunk 2 above)
Comment (klinzo) [reply .circleci/src/jobs/build_ios.yml:66]: Could you help me with that? How would one do that?

Comment (maxammann) [reply .circleci/src/jobs/build_ios.yml:66]: In fastlane there is a environment variable EXTRA_PACKAGER_PARAMS or similar. You can just append there " --sourcemap-output ios/output/integreat.ios.bundle.map"

This works for ios and android

Comment (klinzo) [reply .circleci/src/jobs/build_ios.yml:66]: Ahh found the issue. Afaics, the sourcemap is generated automatically for android. For ios just the path was wrong...

**Review by maxammann — APPROVED** [review]
config.yml is outdated. Created a task to prevent this: https://issues.tuerantuer.org/browse/IGAPP-311

**Pull request merged**